ipatests/test_integration/test_installation.py

```diff
@@ -444,6 +457,64 @@ def test_ipactl_restart_pki_tomcat(self):
         pki_pid_after_restart_2 = get_pki_tomcatd_pid(self.master)
         assert pki_pid_after_restart != pki_pid_after_restart_2
 
+    def test_ipactl_scenario_check(self):
+        """ Test if ipactl starts services stopped by systemctl
+
+        This will first check if all services are running then it will stop
+        few service. After that it will restart all services and then check
+        the status and pid of services.It will also compare pid after ipactl
+        start and restart in case of start it will remain unchanged on the
+        other hand in case of restart it will change.
+        """
+        # listing all services
+        services = ["Directory", "krb5kdc", "kadmin", "named", "httpd",
```

Comment:
Please use proper indention. Since the list does not fit in one line, indent it as:
```
services = [
    "Directory", "krb5kdc", "kadmin", "named", "httpd",
    ...
]
```